modules/gdscript/gdscript_analyzer.cpp

```diff
@@ -5240,6 +5235,20 @@ void GDScriptAnalyzer::is_shadowing(GDScriptParser::IdentifierNode *p_identifier
 		parent = ClassDB::get_parent_class(parent);
 	}
 }
+
+void GDScriptAnalyzer::check_conversion(const GDScriptParser::Node *p_source, const Variant::Type &p_from_type, const Variant::Type &p_to_type) {
+	if (p_from_type == Variant::FLOAT && p_to_type == Variant::INT) {
+		parser->push_warning(p_source, GDScriptWarning::NARROWING_CONVERSION);
+	}
+
+	// We only want to warn about a copy if the use comes from a variable access
```

Comment:
This should probably be expanded to any expressions that access an existing value but I'm not familar with the AST

Reply:
```suggestion
	// We only want to warn about a copy if the use comes from a variable access.
```
Per the [comment style](https://docs.godotengine.org/en/latest/contributing/development/code_style_guidelines.html#comment-style-guide).